templates/spin_sidebar_v2.hbs

```diff
@@ -187,6 +187,10 @@
                                         class="active" {{/if}}
                                         href="{{site.info.base_url}}/spin/v2/managing-plugins">Managing Plugins</a>
                         </li>
+                        <li><a {{#if (active_project request.spin-full-url "/spin/v2/testing-apps" )}}
```

Comment:
I'd put this in the Applications area rather than under Tools.

Reply:
Thanks, this one in post fix.